### ISSUE TITLE
Include Curtain Panels in Material Count

### ISSUE THREAD

**Issue by KBigart** [ThorntonTomasetti/Beacon#6]
I'm having a hard time confirming this... but I believe the program is completely ignoring Curtain Panel construction within the model?

We use curtain panels to model our precast (since they function similar)... and therefore there is a lot of precast concrete Area in our model/panels that seems to be missing from the data. This also would mean that all glazing seems to be missing from the counts as well?

**Comment by KBigart** [comment]
Actually investigating a bit more... this appears to only really be focused on Structural Elements (which i understand). But as an Architectural group, we are looking at all other items... so our metal panel facade, railings, glass, precast, etc... all these elements seem to be missing. 

It'd perhaps be a good enhancement to include a checklist similar to the phases checklist for also Revit categories at the start up of Beacon. That way you can choose what elements to look at (beyond floors/framing/etc). Would also allow really focused investigations for doing specific investigations on "different railing layouts" and seeing the % different just for one isolated category.

**Comment by ktam-tt** [comment]
Right, Beacon is only handling Structural Elements for now. Adding Architectural elements is certainly possible at a later time.
